libraries/webInterface/gui/index.js

```diff
@@ -1571,6 +1568,27 @@ realityServer.gotClick = function (event) {
     }
 };
 
+/**
+ * Verifies that the name of a new object or frame meets certain criteria
+ * (e.g. alphanumeric)
+ * Also shows an error message if the name doesn't qualify for a non-obvious reason
+ * @param textContent
+ * @return {boolean}
+ */
+function isNameOk(textContent) {
+    if (textContent === 'Enter Name') {
+        return false;
+    }
+    if (textContent === '') {
+        return false;
+    }
+    let isAlphanumeric = /^[a-z0-9]+$/i.test(textContent);
+    if (!isAlphanumeric) {
+        showErrorNotification('Name must be alphanumeric');
```

Comment:
It must be lowercase as well according to the regex

Reply:
oops I'll fix the regex to not be lowercase, I think mixed case should be allowed